iavl/cli.py

```diff
@@ -344,5 +356,44 @@ def visualize(db, version, store=None, include_prev_version=False):
     print(g.source)
 
 
+@cli.command()
+@click.option(
+    "--db", help="path to application.db", type=click.Path(exists=True), required=True
+)
+@click.option("--store", "-s")
+@click.option(
+    "--version",
+    help="the version to query, default to latest version if not provided",
+    type=click.INT,
+)
+def state_changes(db, version, store: Optional[str]):
```

Comment:
same [reason](https://github.com/crypto-com/python-iavl/pull/3#discussion_r1034392113) for optional store right?

Reply:
yes